### DISCUSSION TITLE
how can i covert the pth model to tensorrt model by c/c++

### DISCUSSION THREAD

**Discussion by leemayi** [open-mmlab/mmdeploy#1607] [General]
I train the model (. pt) on the Linux machine, and I need to predict on different Windows machines. I know on the Windows that python can export engine models, but this requires installing the python environment. Can you export engine models through the c/c++api

**Comment by lvhan028** [comment]
MMDeploy doesn't support exporting engine models with C/C++.
I think you export engine models on your Linux machine and use them on your windows platform as long as they have the same TensorRT release
